### DISCUSSION TITLE
Regression weight not found issue.

### DISCUSSION THREAD

**Discussion by wcahn** [brianhie/efficient-evolution#8] [Q&A]
Hi! 

I'm trying to run this on my MacBook pro (intel).

just cloned and run bin/recommend.py.

I got results, but errors also occurred.

"...../python3.9/site-packages/esm/pretrained.py:134: Regression weights not found, predicting contacts will not produce correct results."

Some say it is okay, but I just wanna get confirmation from you!

Thank you!

**Comment by brianhie** [comment]
Yup, this is expected behavior, those weights are not needed
